src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/hubandspoke/HubBolt.java

```diff
@@ -57,14 +57,24 @@ public HubBolt(Config config) {
 
     @Override
     protected void handleInput(Tuple input) throws Exception {
-        if (hubConfig.getRequestSenderComponent().equals(input.getSourceComponent())) {
+        if (input.getSourceComponent().equals(hubConfig.getLifeCycleEventComponent())) {
```

Comment:
Is it supposed to call `AbstractBolt.handleLifeCycleEvent` or it's overloaded version? If so, then routing must be done inside `AbstractBolt.dispatch`.